azure-pipelines-official.yml

```diff
@@ -0,0 +1,162 @@
+        os: windows
+      sbom:
+        enabled: false
+    pool:
+      name: NetCore1ESPool-Svc-Internal
+      image: windows.vs2022preview.amd64
+      os: windows
+    customBuildTags:
+    - ES365AIMigrationTooling
+    
+    stages:
+    - stage: build
+      displayName: Build
+      jobs:
+      - template: /eng/common/templates-official/jobs/jobs.yml@self
+        parameters:
+          artifacts:
+            publish:
+              artifacts: true
+              logs: true
+              manifests: true
+          enableMicrobuild: true
+          enablePublishUsingPipelines: true
+          workspace:
+            clean: all
+          jobs:
+          - job: Windows_NT
+            timeoutInMinutes: 90
+            pool:
+              name: netcore1espool-internal
+              image: 1es-windows-2022-pt
+              os: windows
+            strategy:
+              matrix:
+                  Build_Release:
+                    _BuildConfig: Release
+
+            preSteps:
+            - checkout: self
+              clean: true
+            steps:
+            - task: UseDotNet@2
+              displayName: Get .NET Core SDK
+              inputs:
+                version: 8.x
+            - task: DotNetCoreCLI@2
+              displayName: Run Tests
+              inputs: 
+                command: test
+            - script: eng\common\cibuild.cmd
+                -configuration $(_BuildConfig)
+                -projects src/Signing/SignScripts.csproj
+                -prepareMachine
+                $(_InternalBuildArgs)
+                /p:Test=false
+                /p:DotNetSignType=$(_SignType)
+                /p:TeamName=$(_TeamName)
+              displayName: Sign scripts
+              env:
+                SIGNTOOL_DISABLE_TELEMETRY: 'true'
+            - task: PowerShell@2
+              displayName: Validate Signature
+              inputs:
+                filePath: eng/common/sdk-task.ps1
+                arguments: -task SigningValidation -restore -msbuildEngine vs 
+                  /p:InputFiles='$(Build.Repository.LocalPath)/artifacts/bin/SignScripts/**/*.ps1'
+                  /p:PackageBasePath='$(Build.Repository.LocalPath)/artifacts/bin/SignScripts/'
+      - job: LinuxTests
+        timeoutInMinutes: 30
+        pool:
+          name: netcore1espool-internal
+          image: 1es-ubuntu-2204-pt
+          os: linux
+        steps:
+        - checkout: self
+          displayName: Checkout Self
+          clean: true
+        - task: UseDotNet@2
+          displayName: Get .NET Core SDK
+          inputs:
+            version: 8.x
+        - script: |
+            chmod u+x src/dotnet-install.sh
+          displayName: 'Add permission to execute'
+        - task: DotNetCoreCLI@2
+          displayName: Run Tests
+          inputs: 
+            command: test
+
+      - job: MacOSTests
+        timeoutInMinutes: 30
+        pool: 
+          name: Azure Pipelines
+          image: macOS-11
```

Comment:
Similarly this is pretty old, _should_ it be?

Reply:
I have double checked with the 1es team - it's ok to bump it.